src/styles/components/_search-bar.scss

```diff
@@ -3,6 +3,11 @@ $dso-search-bar-height: $u5;
 .dso-search-bar {
   display: flex;
 
+  &.dso-search-bar-small {
+    margin-bottom: $u2;
+    width: 320px - $u4;
```

Comment:
wat is `320px` voor een maat? Kan die niet beter naar een variabele?

Reply:
320px is de allersmalste resolutie waar we rekening mee houden. Maar ik zal er een variabele van maken.